src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/UnallocatedCasesService.kt

```diff
@@ -22,4 +22,9 @@ class UnallocatedCasesService(
     val convictions = communityApiClient.getConvictions(crn)
     return convictions.sortedByDescending { c -> c.convictionDate }.first().sentence.startDate
   }
+
+  fun getInitialAppointmentDate(crn: String, contactsFromDate: LocalDate): LocalDate? {
+    val contacts = communityApiClient.getInductionContacts(crn, contactsFromDate)
+    return contacts.minByOrNull { c -> c.contactStart }?.contactStart?.toLocalDate()
```

Comment:
should we store the time and convert it to localDate when retrieving or follow what I did here? i.e. convert from the delius response and only care about the date?

Reply:
Just checked the prototype and we always only ever care about the date so I think we are good to drop the time here and store only the date